src/IECoreHoudini/plugin/Plugin.cpp

```diff
@@ -98,36 +98,60 @@ void newSopOperator(OP_OperatorTable *table)
 {
 	OP_Operator *opHolder = new OP_Operator(
 		"ieOpHolder", "Cortex Op",
-		SOP_OpHolder::create, SOP_ParameterisedHolder::parameters, 0, 4,
-		SOP_ParameterisedHolder::variables, OP_FLAG_GENERATOR
+		SOP_OpHolder::create,
+		SOP_ParameterisedHolder::parameters,
+#if UT_MAJOR_VERSION_INT >= 16
+		SOP_OpHolder::theChildTableName,  
```

Comment:
https://www.sidefx.com/docs/hdk/_o_p___node_8h_source.html#l00203 The child table name is the type of node that the node can have children of (for example, VOP, SOP, CHOP). It looks like it's required for child nodes to be added.